src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -422,7 +422,7 @@ function BaseSelectionList<TItem extends ListItem>(
                                 />
                             </View>
                         )}
-                        {!!headerMessage && (
+                        {!!headerMessage && !showLoadingPlaceholder && (
```

Comment:
@burczu bump on the comment above

Reply:
@aimane-chnaif You may be right this is not the place for this change and your proposed change does not fix the root issue we want to address with this PR - the message is visible while loading members for some short period. I think I need to find completely different approach to fix this - I'll try to fix it today then.

Reply:
@aimane-chnaif I've changed the approach by introducing new prop in the `BaseSelectionList` component called `notFoundMessage` - this way can threat this special message separately and show it only after loading process is finished (if there is no items loaded). Other messages could show up even while loading, as it was before. 

What do you think?

Reply:
@aimane-chnaif I've tried and the message is shown for some milliseconds no matter what condition I use - I think because the beginning state of the list is loading, only fixing it in this component fixes the issue...